BLSQ/iaso
# Polio 913 add xls template

Allow users to download a template of the exczel they should upload for the "Import line list" feature in polio

Related JIRA tickets :  POLIO-913

## Self proofreading checklist


## Changes

- Fixed bug in `useSnackMutations` that prevented it from handling errors correctly
- Fixed a bug with the line list import that was crashing because the `account` wasn't passed when creating a campaign
- Removed double success snackbar 
- Refactored ÌmportLineListComponent` to `ImportLineList` following the model of `BulkUserImportDialog`. Converted to TS
- refactored `BulkUserImportButtons` to the more generic (and reusable- `FileUpoadButtons`

## How to test

- Go to polio
- Click the Import button
- Click the download template button -> it should download an xls template
- To test that the original feature still works
    - get the example file reference din the ticket
    - select it from the import modal
    - confirm 

## Print screen / video

Uploading Screen Recording 2023-03-27 at 13.42.54.mov…
